package/network/services/dropbear/Config.in

```diff
@@ -49,6 +58,15 @@ config DROPBEAR_ECC_FULL
 
 		Increases binary size by about 4 kB (MIPS).
 
+config DROPBEAR_CHACHA20POLY1305
+	bool "Chacha20-Poly1305 support"
+	default y
+	help
+		This enables the following authenticated encryption mode:
+		  chacha20-poly1305@openssh.com
```

Comment:
Typo?

Reply:
> Typo?

sorry, please clarify, what do you mean by typo? help wording, or default==y or algo name, or?

Reply:
Sorry, shouldn't it say cipher instead of mode?

Reply:
ah, I see. well, to be precise, chacha20 here is the only chiper while poly1305 is authenticator, combined they are aead (authenticated encryption with associated data) mode or just algo for simplicity, another example is aes-gcm.
so, I believe, "cipher" wouldn't be correct/fair enough, and that's why "mode" was used.
but sure can change that, do we need?

Reply:
@diizzyy  unlike [rfc7539](https://tools.ietf.org/html/rfc7539), [openssh's draft](https://tools.ietf.org/html/draft-josefsson-ssh-chacha20-poly1305-openssh-00) refers it as cipher, so your proposal is more than valid, thank you